react/finger-exercises/redux/01 - actions and reducers/books/actions.js

```diff
@@ -23,5 +24,17 @@ export default {
     payload: {
       title
     }
+  }),
+  setFavoriteBook: title => ({
+    type: actions.SET_FAVORITE_BOOK,
+    payload: {
+      favoriteBook: title
```

Comment:
We usually send the payload directly, without an object if its no needed:
```js
{
  type: actions.SET_FAVORITE_BOOK,
  payload: title
}
```

There is no need to change it in this pr, just keep it in mind for the next ones.

Reply:
:+1: